interpreter/cling/lib/UserInterface/UserInterface.cpp

```diff
@@ -66,6 +67,43 @@ namespace {
 
     textinput::TextInput* operator -> () { return &m_Input; }
   };
+
+  llvm::SmallString<512> GetHistoryFilePath() {
+    if (getenv("CLING_NOHISTORY")) {
+      return {};
+    }
+
+    if (const char* HistFileEnvvar{std::getenv("CLING_HISTFILE")}) {
```

Comment:
I find the suggestion below slightly more readable (but up to you to change it) :slightly_smiling_face: 
```suggestion
    if (const char* HistFileEnvvar = std::getenv("CLING_HISTFILE")) {
```

Reply:
Done.